client/src/components/showForm/showForm.module.scss

```diff
@@ -25,3 +25,11 @@
   align-items: center;
   margin: 20px;
 }
+
```

Comment:
Seems like you were testing something. Do these rule sets need to stay in here? They aren't being applied anywhere as far as I can tell.

Reply:
deleted